### PR TITLE
Disable local data picker on ios

### PR DESCRIPTION
**Pull request by m-kuhn**
... until we have something that works

### CODE REVIEW

**Review by nirvn — APPROVED** [review]

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 149d8d03b5fac97040a220650247074339a47b1d: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3193-9218bf7fd9294b2c34bb8d199477d373a4edfa6d-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3193-9218bf7fd9294b2c34bb8d199477d373a4edfa6d-arm-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3193-9218bf7fd9294b2c34bb8d199477d373a4edfa6d-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-3193-9218bf7fd9294b2c34bb8d199477d373a4edfa6d-x86-android.apk)
</details>

File: src/qml/WelcomeScreen.qml [hunk 1]
```diff
@@ -403,6 +403,7 @@ Page {
         }
         QfButton {
           id: localProjectButton
+          visible: Qt.platform.os !== "ios" // iOS does not allow random file access. Disable local data picker until we have a proper import/export workflow like on Android
```
Comment (nirvn) [inline src/qml/WelcomeScreen.qml:406]: I think it's better to disable here than hide, for a few reasons: i) hidden could lead to users misunderstanding we are not planning to support non-cloud projects the way we do on android, and ii) reduces the chance of someone supporting the effort of proper implementation :)

```suggestion
          enabled: Qt.platform.os !== "ios" // iOS does not allow random file access. Disable local data picker until we have a proper import/export workflow like on Android
```

Comment (m-kuhn) [reply src/qml/WelcomeScreen.qml:406]: But: Hidden will also make people think "what did I do wrong"

Comment (nirvn) [reply src/qml/WelcomeScreen.qml:406]: @m-kuhn , how though, if someone isn't familiar with QField on Android, he/she/they will not know there's a potential local dataset support no?

Also, `enabled: false` is what we did to tease the cloud side of things in the QField 1.x cycle.

Comment (m-kuhn) [reply src/qml/WelcomeScreen.qml:406]: Disabling the button will help them understand it's not available right now but with now hint what they can do to fix the problem which leaves it as "broken functionality cluttering my screen and the easiest thing I can do is giving a bad review that functionality is borken"?

**Pull request merged**